templates/download/risc-v/tab-1.html

```diff
@@ -2,6 +2,16 @@
 	<div class="row u-vertically-center u-no-padding">
 		<div class="col-6">
 			<h2>Download Ubuntu Server Preinstalled Image</h2>
+			<h3 class="p-heading--4">Ubuntu Server 22.10</h3>
+			<div class="p-notification--information is-inline">
+				<div class="p-notification__content">
+					<h4 class="p-notification__title">Note:</h4>
+					<p class="p-notification__message">It is an early RISC-V developer access through Ubuntu 22.10 and this is a preinstalled image, if you have a NVMe drive, we advise you to use the Ubuntu Server live installer which can be found <a href="https://cdimage.ubuntu.com/releases/22.04.1/release/ubuntu-22.04.1-preinstalled-server-riscv64+unmatched.img.xz">here</a>.</p>
+				</div>
```

Comment:
This link has already been provided

Reply:
That is not the link we are waiting for. It is the link under 'How to install Ubuntu on the LicheeRV Dock'